spec/dummy/config/initializers/fae.rb

```diff
@@ -12,6 +12,7 @@
     ja: 'Japanese'
   }
 
+  # has_top_nav has been deprecated, but keeping in to ensure definition doesn't rasie exceptions
```

Comment:
`raise`. Do you want to note what version it's officially disappearing from? `@depreciation - ... in vX` is how I did it in the JS/SCSS/Slim. Makes it easy to search for in the code before the version is released.